.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -303,24 +303,24 @@ jobs:
     - name: List excluded SYCL CTS categories
       if: inputs.tests_selector == 'cts' && env.CTS_TESTS_TO_BUILD == ''
       run: |
+        cts_exclude_filter=""
         if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_OCL_CPU
-        fi
-        if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
-          cat $PWD/devops/cts_exclude_filter_L0_GPU
+          cts_exclude_filter=cts_exclude_filter_OCL_CPU
+        elif [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
+          cts_exclude_filter=cts_exclude_filter_L0_GPU
         fi
+        cat $PWD/devops/$cts_exclude_filter
     - name: Build SYCL CTS tests
       if: inputs.tests_selector == 'cts'
       env:
         CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
       run: |
-        # Do not exclude any category if passing the CTS_TESTS_TO_BUILD variable.
         cts_exclude_filter=""
+        # If CTS_TESTS_TO_BUILD is null - use filter
         if [ -z "$CTS_TESTS_TO_BUILD" ]; then
           if [ "${{ contains(inputs.target_devices, 'opencl:cpu')  }}" = "true" ]; then
             cts_exclude_filter=$PWD/devops/cts_exclude_filter_OCL_CPU
-          fi
-          if [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
+          elif [ "${{ contains(inputs.target_devices, 'level_zero:gpu')  }}" = "true" ]; then
```

Comment:
> @aelovikov-intel is it what you were https://github.com/intel/llvm/pull/14325#discussion_r1691818740 about?

No, we still have duplication here (same as line 309).

Reply:
What about just merging these two steps into one?

Reply:
That's what I initially suggested:
> IMO, we should drop this step and print that information after we've selected the right file.

Reply:
lol, a bit of attentivenness:D done in [b8d9777](https://github.com/intel/llvm/pull/14807/commits/b8d9777cdf5b5db4c1a2f0f4dfc708d1027ba1d9)